README.md

```diff
@@ -30,7 +30,10 @@ aedesPersistenceRedis({
   family: 4,           // 4 (IPv4) or 6 (IPv6)
   password: 'auth',
   db: 0,
-  maxSessionDelivery: 100 // maximum offline messages deliverable on client CONNECT, default is 1000
+  maxSessionDelivery: 100, // maximum offline messages deliverable on client CONNECT, default is 1000
+  packetTTL: function (packet) { // offline message TTL, default is 30 days
+    return 10 //seconds
+  }
```

Comment:
why do we need this customizable for every single packet? i would have put that as a constant.